qutebrowser/config/configtypes.py

```diff
@@ -1223,14 +1224,68 @@ class AcceptCookies(BaseType):
                                ('never', "Don't accept cookies at all."))
 
 
-class ConfirmQuit(BaseType):
+class ConfirmQuit(List):
 
     """Whether to display a confirmation when the window is closed."""
 
+    typestr = 'string-list'
+
     valid_values = ValidValues(('always', "Always show a confirmation."),
                                ('multiple-tabs', "Show a confirmation if "
                                                  "multiple tabs are opened."),
+                               ('downloads', "show a confirmation if downloads"
+                                             "are running"),
                                ('never', "Never show a confirmation."))
+    # Values that can be combined with commas
+    combinable_values = ('multiple-tabs', 'downloads')
+
+    def transform(self, value):
+        # Backward compatible
```

Comment:
Is there any good reason to stay backwards-compatible here? Of course the config from an user-facing view should be, but what value we get from `config.get(...)` can change as long as all places are changed as well - which is the case here.

So I'd prefer to not override `transform` and drop the `isinstance(values, list)` checks for much clearer code.